src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_utils.py

```diff
@@ -449,3 +449,17 @@ def interrupt_main_thread(main_thread):
         except:
             pydev_log.exception('Error on interrupt main thread fallback.')
 
+
+def current_thread():
```

Comment:
`threading.current_thread` is already available in Python 2.6, so, I don't think we actually need this helper function. If an alias is interesting, I think it should be in `pydevd_constants`, right after the `threading` module is imported.